index.html

```diff
@@ -310,29 +417,30 @@ <h2 id="connect">Let's Connect</h2>
               <i class="fa-solid fa-chevron-right"></i>
             </li>
           </ul>
-        </li>
-      </ul>
+        </div>
+      </article>
     </section>
+    <!-- CONTACT SECTION -->
     <section id="contact-form-container">
-      <ul id="contact-form-container-ul">
-        <li>
+      <article id="contact-form-container-article">
+        <div>
           <h2 id="contact-form-header">Contact me</h2>
-        </li>
-        <li>
+        </div>
+        <div>
           <p id="contact-form-text">
             If you have an application you are interested in developing, a
             feature that you need built or a project that needs coding. I’d love
             to help with it
           </p>
-        </li>
-        <li>
+        </div>
+        <div>
           <form
```

Comment:
- [ ] [OPTIONAL] In a bid to further improve your design, I think reducing the `gap` or `space` in-between the input boxes will be a nice move. It seems too much farther from the Figma design. Also, You may consider increasing the `font-size` and `width` of your action button - `Get in touch` so that it can be more visible and clickable to/by users.

- _See what I meant: :point_down: :point_down:_

|Your Design| Figma Design|
|:--------------:|:-----------------:|
|![image](https://user-images.githubusercontent.com/99715640/236638587-a45a8e50-b8b4-4d93-93ec-e417433994f2.png)|![image](https://user-images.githubusercontent.com/99715640/236638596-800cfa19-7533-41a7-a5dc-d95bcf4fef76.png)|